x/launch/keeper/msg_request_add_validator.go

```diff
@@ -67,6 +67,9 @@ func (k msgServer) RequestAddValidator(
 
 		err = ctx.EventManager().EmitTypedEvent(&types.EventValidatorAdded{
 			GenesisValidator: *content.GetGenesisValidator(),
+			LaunchID:         msg.LaunchID,
+			HasCampaign:      chain.HasCampaign,
+			CampaignID:       chain.CampaignID,
```

Comment:
Would it be possible to also emit:
1. New validator operator address.
2. The address of the account which approved this validator (coordinator)